packages/integration_test/lib/common.dart

```diff
@@ -121,3 +132,105 @@ class Failure {
     return Failure(failure['methodName'], failure['details']);
   }
 }
+
+/// Integration web tests can execute WebDriver actions such as screenshots.
+///
+/// These test will use [TestStatus] to notify `integration_test` of their
+/// state.
+enum TestStatus {
+  /// Test is waiting for executing WebDriver actions.
+  waitOnWebdriverCommand,
+
+  /// Test executed the previously requested action.
+  webdriverCommandComplete,
+}
+
+/// Types of different WebDriver actions that can be used in web integration
+/// tests.
+///
+/// These actions are either commands that WebDriver can execute or used
+/// for the communication between `integration_test` and the driver test.
+enum WebDriverActionTypes {
+  /// Acknowlegement for the previously sent message.
+  ack,
+
+  /// No further WebDriver Action is necessary.
+  noop,
+
+  /// Asking WebDriver to take a screenshot of the Web page.
+  screenshot,
+}
+
+/// Command for WebDriver to execute.
+///
+/// Only works on Web when tests are run via `flutter driver` command.
+///
+/// See: https://www.w3.org/TR/webdriver/
+class WebDriverAction {
```

Comment:
I would recommend that we not use the term "Action" to avoid confusion.

The WebDriver spec refers to taking screenshots as a "command", not "action" (see https://www.w3.org/TR/webdriver/#screen-capture). "Action" seems to have a different meaning for WebDriver: https://www.w3.org/TR/webdriver/#actions. In fact, the dartdoc for this class also call it "Command".

Reply:
done. renamed all the files.